deploy/k8s/kustomize/base/cloudbeat-ds.yml

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
```

Comment:
why do we need to change this file?

Reply:
Because we use it as part of our flow, and it should correspond.